etc/profile-m-z/youtube-dl-gui.profile

```diff
@@ -0,0 +1,58 @@
+# Firejail profile for youtube-dl-gui
+# Description: A cross platform front-end GUI of the popular youtube-dl media downloader
+include youtube-dl-gui.local
+# This file is overwritten after every install/update
+include globals.local
+
+#These are blacklisted by disable-interpreters.inc
+include allow-python2.inc
+include allow-python3.inc
+
+noblacklist ${HOME}/.config/youtube-dlg
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/youtube-dlg
+whitelist ${HOME}/.config/youtube-dlg
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+# Following degrades performances
+#caps.drop all
+#machine-id
+#netfilter
+#no3d
```

Comment:
```suggestion
caps.drop all
machine-id
netfilter
# Following degrades performances
#no3d
```

Only `no3d` degrades performances.
`no3d` `blacklist`s [`/dev/dri`](https://en.wikipedia.org/wiki//dev/dri), therefore ffmpeg falls back to software en/decoding instead of hardware accelerated en/decoding.

Reply:
oh.. then no need for no3d.